src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -212,10 +176,12 @@ def setUp(self):
                 self.hostlist_clients, self.workdir,
                 self.hostfile_clients_slots)
 
-        self.agent_sessions = agent_utils.run_agent(
-            self.basepath, self.hostlist_servers, self.hostlist_clients)
+        # Start agent and server
+        if self.setup_start_agents == True:
+            self.agent_sessions = agent_utils.run_agent(
+                self.basepath, self.hostlist_servers, self.hostlist_clients)
 
-        if self.setup_start_servers:
+        if self.setup_start_servers == True:
```

Comment:
(pylint-singleton-comparison) Comparison to True should be just 'expr' or 'expr is True'